meta-balena-common/classes/sign-efi.bbclass

```diff
@@ -0,0 +1,44 @@
+inherit deploy
+
+# Do not run on native recipes
+do_sign_efi:class-native() {
+    :
+}
+
+do_sign_efi () {
+    if [ "x${SIGN_API}" = "x" ]; then
```

Comment:
This should also have a check for the `SIGN` variable which is the global flag for enabling/disabling all signing. We want to keep the possibility to build an unsigned release while keeping the metadata defined. Something like
```
if [ "${SIGN}" != "true" ]; then
    bbnote "Signing disabled, nothing to do"
    exit 0
fi
```

Reply:
We will lock this to `SIGN_API` and ditch `SIGN`